text/0010-gatsby-docs-localization

```diff
@@ -0,0 +1,104 @@
+* Start Date: 2019-09-11
+* RFC PR:
+* Gatsby Issue:
+
+## Summary
+
+Begin the process of localizing the documentation of Gatsby into non-English languages.
+
+## Motivation
+
+[You Belong Here.](https://www.gatsbyjs.org/blog/2018-09-07-gatsby-values/#you-belong-here)
+
+Gatsby believces that everyone belongs in Open Source, and that includes developers that do not speak English. Having access to translated documentation opens up the ability to create websites to people who don't speak English and don't have the privileges to be able to learn.
+
+## Detailed Design
+
+There are two main approaches we can take for the translations. Whichever path we choose depends on the prorities of the Gatsby team, hence the Request for Comments. The options are: (1) doing translations on GitHub directly or (2) using a translation SaaS platform such as Crowdin. This document will go over both alternatives and the advantages and drawbacks of each option.
+
+## Alternative 1: GitHub-based translations
+
+The first and simplest option is to simply create and host the translations on GitHub itself, along with the rest of the Gatsby documentation. For smaller websites (like [Dan Abramov's blog](https://github.com/gaearon/overreacted.io)) it's enough to put each translation in a markdown file next to the source file and use `gatsby-i18n`.
+
+For larger websites, especially documentation websites that have lots of PRs and require versioning and up-to-date content, this isn't enough. The docs for [Angular](https://github.com/angular/angular-cn/tree/aio), [Vue](https://github.com/vuejs/jp.vuejs.org), and [React](https://github.com/reactjs/es.reactjs.org) store their translations in copies of the repo.
+
+### Advantages
+
+This was the approach that we took for translating the React docs. It appealed to us for a lot of reasons:
+
+* Open Source Contributors tend to be familiar with Git
+* No need to integrate with an external API and software (aside from GitHub)
+* People want to be listed as contributors of the React/Gatsby organization on GitHub, and it's an incentive to contribute.
+* Can use GitHub features such as Code Owners and pull requests as quality assurance
+
+### Drawbacks
+
+The main drawback of this approach is that the Gatsby docs currently live in a monorepo with the rest of the Gatsby source, which means we can't just do a copy of the entire website and use `git merge` like we do with React (and Vue).
```

Comment:
We do similar copying out of directories for our starters — our bot checks on each commit to master if there's any changes to a starter and then commits that — we could use the same logic to create PRs to language repos — https://github.com/gatsbyjs/gatsby-starter-default

Reply:
Oh interesting!

So we could probably do something like:

1. Create a copy of the `docs/` directory in a new repo in a new language
2. Update that directory whenever there are changes to a file in the original repo
3. Copy the results of those repos into a single readonly `gatsby-i18n` repo that is built and deployed as the actual gatsby website

Reply:
Hi! In Crowdin you can use one repo for a different project and with the GitHub integration you can specify which files should be uploaded for localization and which ones should be ignored. No need to make any copies and bother with merging. Also our team is ready to help with the setup 24/7 :)